data/uhc_pack/functions/running/diamond_chestplate/on_obtain.mcfunction

```diff
@@ -0,0 +1,5 @@
+advancement revoke @s only uhc_pack:running/obtain_diamond_chestplate
+
+# Mark the player who received the chestplate on a scoreboard.
+# This means cleanup can be handled the tick after, which eliminates desync issues
```

Comment:
Interesting. I don't remember having this problem with the `suspicious_stew` detection.

Reply:
Yeah, when I tried it out with doing the give and clear the same tick, the chestplate became visually glitched. The player could still technically equip it. After opening the inventory again it'd suddenly vanish, this was be the most reliable workaround I could find :P